changelog.md

```diff
@@ -170,4 +170,15 @@
 - ``experimental`` is now a pragma / command line switch that can enable specific
   language extensions, it is not an all-or-nothing switch anymore.
 
+- Nintendo Switch was added as a new platform target. Simply add --os:nintendoswitch
+  to your usual ``nim c`` or ``nim cpp`` command. DevkitPro setup must be the same as
```

Comment:
This is great! But I think a better place for it is in the ``nimc`` docs: https://github.com/nim-lang/Nim/blob/devel/doc/nimc.rst

You can link to them from the changelog :)

Reply:
Cool, sounds good. I'll make that change.